lona/client/_lona/input-events.js

```diff
@@ -85,6 +85,7 @@ export class LonaInputEventHandler {
 
         node.onclick = function(event) {
             event.preventDefault();
+            event.stopPropagation();
```

Comment:
This may be unexpected if someone uses custom js. Maybe need to add some comment to docs?

Reply:
You are right. I already noted that for the next release. I will add a note to https://lona-web.org/1.x/end-user-documentation/html.html#firing-custom-input-events in the release commit